tox.ini

```diff
@@ -17,6 +17,16 @@ extras = testing
 passenv =
 	windir  # required for test_pkg_resources
 
+[testenv:integration]
+deps = {[testenv]deps}
+extras = {[testenv]extras}
+passenv =
+	{[testenv]passenv}
+	DOWNLOAD_PATH
+commands =
+	pytest --integration {posargs:-vv --durations=10 --no-cov setuptools/tests/integration}
```

Comment:
By pointing to the specific `setuptools/tests/integration` directory by default we avoid time collecting/analysing the other tests (and also avoid displaying a lot of skips in the pytest output)